app/controllers/api/conversion_hosts_controller.rb

```diff
@@ -1,5 +1,18 @@
 module Api
   class ConversionHostsController < BaseController
     include Subcollections::Tags
+
+    # Interface for the polymorphic resource, which could be either a Vm or
+    # a Host. Failure to provide a conversion host ID will raise an error.
+    #
+    # POST /api/conversion_hosts/1 {'action': 'resource'}
+    #
+    def resource_resource(type, id = nil, _data = nil)
```

Comment:
you are raising on `nil id`, do we need defaulting `id` to bi `nil`?

Reply:
I was sort of copying what some of the other code does, I wasn't sure under what circumstance that could happen. I can remove it.